cves/kernel/CVE-2014-3183.yml

```diff
@@ -56,6 +67,9 @@ description_instructions: |
   Your target audience is people just like you before you took any course in
   security
 description:
+  Heap overflow in handling of HID reports sent to Logitech Unifying receivers
```

Comment:
This needs to be less technical and explain more of the jargon that are in these sentences

Reply:
I rewrote the description to provide more info.